modules/payloads/stagers/python/reverse_tcp_ssl.rb

```diff
@@ -1,5 +1,5 @@
 ##
-# This module requires Metasploit: http//metasploit.com/download
+# This module requires Metasploit: https://www.metasploit.com/download
```

Comment:
The rest are failing the travis build CI. How do I run the build test in the local system before committing?